README.md

```diff
@@ -32,6 +32,7 @@
     - [Train the model in the cloud](#train-the-model-in-the-cloud)
   - [📜 Test the newly trained model](#-test-the-newly-trained-model)
   - [🍴 Serve the newly trained model](#-serve-the-newly-trained-model)
+- [🤩 Install latest instructLab upgrades](#-upgrade-instructlab-to-latest-version)
```

Comment:
How about using an 🚀 for representing upgrades?

Reply:
Sure, it looks better.